components/org.wso2.carbon.identity.organization.management.service/src/main/java/org/wso2/carbon/identity/organization/management/service/OrganizationManagerImpl.java

```diff
@@ -855,10 +855,15 @@ private TenantMgtService getTenantMgtService() {
         return OrganizationManagementDataHolder.getInstance().getTenantMgtService();
     }
 
-    private void validateOrganizationAllowedToAccess(String organizationId) throws OrganizationManagementException {
+    private void validateOrganizationAllowedToAccess(String organizationId, boolean allowSameOrg)
```

Comment:
Shall we add a method comment to this, explaining only ancestors of an organization are allowed to manage the orgs, and if self-manage is allowed `allowSameOrg` param need to be set as true

Reply:
In order to make this method self explainery shall we change the method signature

`validateOrganizationAccess(String requestInvokingOrganizationId, String accessedOrganizationId, boolean isSelfOrgManageAllowed)`

Reply:
Fixed by - https://github.com/wso2-extensions/identity-organization-management-core/pull/4/commits/7db20455d584144ca94284a12aa7c9581104b471